public/js/views/management.jsx

```diff
@@ -70,14 +54,9 @@ export default class Management extends Component {
     this.props.getUserSubscriptions();
   }
 
-  userSignIn = e => {
-    e.preventDefault();
-    this.props.userSignIn();
-  }
-
-  userSignOut = e => {
+  showSignOut = e => {
     e.preventDefault();
-    this.props.userSignOut();
+    this.props.showSignOut();
```

Comment:
the thing I changed it to is more like you navigate to a sign out page. For that reason I followed the pattern we already have of `showNewPage()`.